src/containers/my-cooperations/cooperation-details/CooperationDetails.tsx

```diff
@@ -57,6 +59,10 @@ const CooperationDetails = () => {
     onResponseError: responseError
   })
 
+  useEffect(() => {
+    response.sections && response.sections.length && setEditMode(true)
```

Comment:
Maybe use `if` here for better readability?

Reply:
as for me this approach better